coderd/database/queries/provisionerjobs.sql

```diff
@@ -162,11 +182,11 @@ SELECT
 	w.id AS workspace_id,
 	COALESCE(w.name, '') AS workspace_name
 FROM
+	final_jobs fj
+INNER JOIN
 	provisioner_jobs pj
-LEFT JOIN
-	queue_position qp ON qp.id = pj.id
-LEFT JOIN
-	queue_size qs ON TRUE
+		ON fj.id = pj.id -- Ensure we retrieve full details from `provisioner_jobs`.
+						 -- JOIN with pj is required for sqlc.embed(pj) to compile successfully.
```

Comment:
Nit: This comment feels a bit misplaced. In general I'd suggest avoiding trailing comments unless they're short and concise and explain that line specifically. Perhaps something more like this:

```sql
-- Join in job data so we can sqlc.embed [...]
JOIN
	provisioner_jobs pj
...
```

Although IMO the comment could be omitted entirely. It's basically an explanation for the `SELECT` which is way up.